cmd/juju/application/deploy_test.go

```diff
@@ -1291,7 +1291,7 @@ func (s *DeploySuite) TestInvalidSeriesForModel(c *gc.C) {
 	withCharmDeployable(s.fakeAPI, curl, "bionic", charmDir.Meta(), charmDir.Metrics(), false, false, 1, nil, nil)
 
 	err := s.runDeployForState(c, charmDir.Path, "portlandia", "--series", "kubernetes")
-	c.Assert(err, gc.ErrorMatches, `cannot add application "portlandia": series "kubernetes" in a non container model not valid`)
+	c.Assert(err, gc.ErrorMatches, `cannot add application "portlandia": \"portlandia\" is not an IAAS charm`)
```

Comment:
If you look at https://github.com/juju/juju/pull/12555, it just uses "container" to differentiate from machine/VM charms. We shouldn't be surfacing CAAS/IAAS to the user.